images/manageiq-ui-worker/Dockerfile

```diff
@@ -18,3 +18,5 @@ RUN chgrp root /var/run/httpd && chmod g+rwx /var/run/httpd && \
 RUN sed -i '/^Listen 80/d' /etc/httpd/conf/httpd.conf
 
 COPY container-assets/manageiq-http.conf /etc/httpd/conf.d
+
+RUN rm -rf /tmp/rpms /create_local_yum_repo.sh /etc/yum.repos.d/local_rpm.repo
```

Comment:
Why do we clean up here, but not in the base Docker file?

Reply:
Explained in 2nd 'Note' in the PR description.  I can change for both manageiq-base and manageiq-ui-worker to do the same local rpm steps (copy rpm, install createrepo, disable default ones and cleanup).